### PR TITLE
Immutable probe queue

### PR DESCRIPTION
**Pull request by hutnikau**
Added temporary immutable queue array to be able find items after flush.

### CODE REVIEW

**Comment by krampstudio** [comment]
Feel free to re-assign the PR (I'll be in holidays next weeks, :palm_tree: )

File: views/js/runner/probeOverseer.js [hunk 1]
```diff
@@ -131,8 +134,8 @@ define([
                 };
                 var args = slice.call(arguments);
                 overseer.getQueue().then(function(queue){
```
Comment (jsconan) [inline views/js/runner/probeOverseer.js]: Not sure you still need to load the storage as you do not take of the result.

**Comment by hutnikau** [comment]
@jsconan I have updated branch in accordance with your comments and resolved conflicts. Please review it again.

File: manifest.php [hunk 2]
```diff
@@ -34,7 +34,7 @@
     'label' => 'Test core extension',
 	'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '3.1.0',
```
Comment (jsconan) [inline manifest.php]: As it is a feature, it should be a minor version change, not a patch version change:
`3.1.0` => `3.2.0`

File: manifest.php [hunk 3]
```diff
@@ -34,7 +34,7 @@
...
+    'version' => '3.1.1',
```
Comment (jsconan) [inline manifest.php]: As it is a feature, it should be a minor version change, not a patch version change:
`3.1.0` => `3.2.0`

**Comment by hutnikau** [comment]
@jsconan I've updated version and changed dependency in this PR:
https://github.com/oat-sa/extension-tao-act/pull/345/files

**Pull request merged**